Corona-Warn-App/src/test/java/de/rki/coronawarnapp/datadonation/OneTimePasswordTest.kt

```diff
@@ -0,0 +1,16 @@
+package de.rki.coronawarnapp.datadonation
+
+import io.kotest.matchers.shouldBe
+import org.junit.Test
+import java.util.UUID
+
+class OneTimePasswordTest {
+
+    @Test
+    fun test() {
+        "923c977a-617b-11eb-ae93-0242ac130002".apply {
+            OneTimePassword(UUID.fromString(this))
+                .payloadForRequest shouldBe encodeToByteArray()
```

Comment:
What is `encodeToByteArray`? Are we testing it too somewhere?

I'd prefer a hardcoded comparison to remove any error sources, e.g.
Find out what that UUID looks like in base64/hex encoded form and use it for comparison.